packages/ffe-grid-react/src/GridCol.js

```diff
@@ -9,7 +9,7 @@ import {
     string,
 } from 'prop-types';
 import classNames from 'classnames';
-import backgroundColors from './background-colors';
+import backgroundColors, { removedColours } from './background-colors';
```

Comment:
I think we use the American spelling of color in FFE. Keeping this consistent will also improve my own sanity. ;-*

Reply:
Actually no, we use `grey` over `gray` already but if that's what it takes to get this thing off my TODO list I'll change it.

Reply:
Well, I agree we should be consistent on that level as well - but we should _at least_ use the same spelling of the same word :P